synapse/handlers/message.py

```diff
@@ -257,6 +257,12 @@ def __init__(self, hs):
         self._block_events_without_consent_error = (
             self.config.block_events_without_consent_error
         )
+        # Some rooms should be excluded from dummy insertion, for instance rooms
+        # without local users who can send events into the room.
+        self._rooms_to_exclude_from_dummy_event_insertion = {}
```

Comment:
please can you document what the structure of this field is? what are the keys, what are the values?